mmv1/products/biglake/Table.yaml

```diff
@@ -0,0 +1,142 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "Table"
+description: |
+  Represents a table.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_tables"
+  api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs.databases.tables"
+base_url: "{{database}}/tables"
+self_link: "{{database}}/tables/{{name}}"
+create_url: "{{database}}/tables?tableId={{name}}"
+update_verb: :PATCH
+update_mask: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  custom_import: templates/terraform/custom_import/biglake_table.go.erb
```

Comment:
Instead of using a custom_import, it should be possible to use an import_format that includes a `%` in the parent field name. basically the same change as I ended up making here: https://github.com/GoogleCloudPlatform/magic-modules/pull/8790/commits/f91638889ce1920976b7ee1b8413c52856965b3e

Reply:
done.